CHANGELOG.md

```diff
@@ -1,6 +1,14 @@
 Change log
 -----------
 
+# v2.0.4+rev2 - 2017-05-28
```

Comment:
Sure. But you could have one release with both meta-resin bump and uboot adaptions. That's what I would do.